test/tests/create-razzle-app.test.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * @jest-environment node
+ */
+'use strict';
+
+const shell = require('shelljs');
+const util = require('../fixtures/util');
+const path = require("path");
+
+shell.config.silent = false;
+
+const stageName = 'stage-cra';
+const craPath = path.join('../node_modules/.bin/create-razzle-app');
+const testPackage = 'razzle-example-basic';
+const testRepo = 'https://github.com/fivethreeo/razzle-example-basic';
+
+describe('create-razzle-app', () => {
```

Comment:
can we add a test and check if `node_modules` directory is available or not
so basically when we run the command with `--no-install` we expect that node_modules folder is empty (or it's not get created from first place) we can also check for `yarn.lock` or `package-lock.json` file ...

Reply:
Maybe just do that to one test? Npm takes so much time to run.

Reply:
yes yes yes